### PR TITLE
Update-Node 2.0

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Update node on steroids ✨

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
Output (sans les couleurs):

```

⬆️  About to bump node version:
- Write .nvmrc
- Write package.json
- Write .travis.yml
+ Successfully bumped Node version to v8.15.1
   fatal: A branch named 'update-node-v8.15.1' already exists.
   Deleted branch update-node-v8.15.1 (was 71299ff).
   Switched to a new branch 'update-node-v8.15.1'
   M    .travis.yml
   [update-node-v8.15.1 ba93d93] Upgrade Node to v8.15.1
    1 file changed, 1 insertion(+), 2 deletions(-)
   Switched to branch 'develop'
   Your branch is ahead of 'origin/develop' by 4 commits.
     (use "git push" to publish your local commits)
+ Commit files 💾
   To https://github.com/Coorpacademy/update-node.git
    + 71299ff...ba93d93 update-node-v8.15.1 -> update-node-v8.15.1 (forced update)
+ Push files on update-node-v8.15.1 📡
  - 📬  Create/Update pull request
  - 🏷  Added label
  - 👥  Create assignations @undefined
+ Successfully handled pull request (#44)
  - 📎  https://github.com/CoorpAcademy/update-node/pull/44
  - 🔖  60eb1f8
  - 🌳  update-node-v8.15.1


⬆️  About to bump depencies cluster core:
+ Successfully updated 1 dependencies of cluster core:
  - bluebird: 3.5.1 -> 3.5.4
+ Updating dependencies lock with yarn 🔐 :
   yarn install v1.12.3
   [1/4] Resolving packages...
   [2/4] Fetching packages...
   [3/4] Linking dependencies...
   warning "@coorpacademy/eslint-plugin-coorpacademy > eslint-plugin-flowtype-errors@3.5.1" has…
    … unmet peer dependency "flow-bin@>=0.66.0".
   warning "@coorpacademy/eslint-plugin-coorpacademy > eslint-plugin-import@2.9.0" ha…
    …s incorrect peer dependency "eslint@2.x - 4.x".
   warning "@coorpacademy/eslint-plugin-coorpacademy > eslint-plugin-mocha@4.12.1" ha…
    …s incorrect peer dependency "eslint@^2.0.0 || ^3.0.0 || ^4.0.0".
   warning "@coorpacademy/eslint-plugin-coorpacademy > eslint-plugin-react@7.4.0" has…
    … incorrect peer dependency "eslint@^3.0.0 || ^4.0.0".
   [4/4] Building fresh packages...
   warning Ignored scripts due to flag.
   success Saved lockfile.
   Done in 5.33s.
   Updated Locks
   fatal: A branch named 'update-core' already exists.
   Deleted branch update-core (was ed054b3).
   Switched to a new branch 'update-core'
   M    package.json
   M    yarn.lock
   [update-core b2e4342] Update core dependencies
    2 files changed, 5 insertions(+), 5 deletions(-)
   Switched to branch 'develop'
   Your branch is ahead of 'origin/develop' by 4 commits.
     (use "git push" to publish your local commits)
+ Commit files 💾
   To https://github.com/Coorpacademy/update-node.git
    + ed054b3...b2e4342 update-core -> update-core (forced update)
+ Push files on update-core 📡
  - 📬  Create/Update pull request
  - 🏷  Added label
  - 👥  Create assignations @undefined
+ Successfully handled pull request (#41)
  - 📎  https://github.com/CoorpAcademy/update-node/pull/41
  - 🔖  60eb1f8
  - 🌳  update-core


⬆️  About to bump depencies cluster testing-tools:
+ No dependencies to update were found

⬆️  About to bump depencies cluster eslint:
+ No dependencies to update were found

Update-node run with success 📤
- update-core: https://github.com/CoorpAcademy/update-node/pull/41

```

File: src/index.js [hunk 1]
```diff
@@ -101,7 +101,10 @@ const commitAndMakePullRequest = config => async ({branch, message}) => {
     config.token
   );
   if (!status.commit) process.stdout.write('+ Did not made a Pull request, nothing has changed\n');
-  else process.stdout.write('+ Successfulley handled pull request\n');
+  else {
+    process.stdout.write('+ Successfully handled pull request\n');
+console.log(status.pullRequest)
```
Comment (AdrieanKhisbe) [inline src/index.js]: oupsiieee

File: package.json [hunk 2]
```diff
@@ -12,20 +16,37 @@
     "update-node": "./src/index.js"
   },
   "scripts": {
-    "test": "exit 0"
+    "test": "npm run lint && npm run test:unit",
```
Comment (Volubyl) [inline package.json:19]: :+1:

File: src/core/config.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+const path = require('path');
+const Promise = require('bluebird');
+const _ = require('lodash/fp');
+const shortstop = require('shortstop');
+const handlers = require('shortstop-handlers');
+
+const resolver = shortstop.create();
+resolver.use('env', handlers.env());
+resolver.use('base64', handlers.base64());
+const readConfig = pathe => JSON.parse(fs.readFileSync(pathe, 'utf-8'));
```
Comment (Fausto95) [inline src/core/config.js:11]: path

Comment (AdrieanKhisbe) [reply src/core/config.js:11]: `path` is already defined 😉

File: package.json (hunk 2 above)
Comment (AdrieanKhisbe) [reply package.json:19]: merci @jfmengels

**Review by Fausto95 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (jfmengels) [reply package.json:19]: Mais de rien !

**Review by godu — APPROVED** [review]

**Pull request merged**